packages/web-twig/README.md

```diff
@@ -155,5 +155,11 @@ Check your Symfony console or log to see if you are using any of the deprecated
 
 ![Deprecations in Symfony's console](https://github.com/lmc-eu/spirit-design-system/blob/main/static/deprecations-symfony-console.png?raw=true)
 
+[components-directory]: https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web-twig/src/Resources/components
+[contributing]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-twig/CONTRIBUTING.md
+[example-guide]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-twig/docs/extendComponents.md
+[changelog]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-twig/CHANGELOG.md
```

Comment:
🔤 OCD triggered 😄 
Still don't understand, why the English alphabet doesn't have `ch`, such a pretty character.